### PR TITLE
Added Activity class structure

### PR DESCRIPTION
**Pull request by piknikki**
*What does this PR do?*

**Clearly and concisely describe the feature.**
- Adds the basic structure for the Activity class

**Is this a feature or refactor?**
- feature

**Problems Encountered & Resulting Solutions**
- none

**How should this be tested?**
- n/a -- no actual functionality yet.

**What steps need to be taken to confirm functionality?**
- n/a

Any questions and possible steps moving forward.
- Continue to fill in the methods as we go.

### CODE REVIEW

File: activity.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Activity {
+  constructor(category, description, minutes, seconds, completed, id) {
+    this.category = category
+    this.description = description
+    this.minutes = minutes
+    this.seconds = seconds
+    this.completed = completed
+    this.id = id
```
Comment (Cody-Durham) [inline activity.js]: can you please add ";" to the ends of lines 3-8

Comment (piknikki) [reply activity.js]: oops!! sorry!!

**Review by Cody-Durham — APPROVED** [review]
Thank you for making the changes!!

**Comment by Cody-Durham** [comment]
Added semicolons to ends of constructor function.

**Pull request merged**